README.adoc

```diff
@@ -121,3 +119,16 @@ With our 3.2.0 release, we added rules checking for adherence to our naming conv
 As of version 3.2.1, we have started adding security-related rules to our plugin:
 
 * Y1. https://github.com/devonfw/sonar-devon4j-plugin/blob/master/src/main/java/com/devonfw/ide/sonarqube/common/impl/check/security/DevonUcImplSecurityConstraintCheck.java[DevonUcImplSecurityConstraintCheck]
+
+== devonfw Java Quality Profile
+
+This plugin comes with a quality profile containing not only the aforementioned rules, but also rules from other plugins that are vital to ensure optimal code quality. Make sure to import the following plugins into your SonarQube instance, so that you are able to fully use this profile:
+
+* https://github.com/checkstyle/sonar-checkstyle/releases/tag/4.27[Checkstyle]
+* https://github.com/QualInsight/qualinsight-plugins-sonarqube-smell/releases/tag/qualinsight-plugins-sonarqube-smell-4.0.0[Code Smells]
+* https://github.com/spotbugs/sonar-findbugs/releases/tag/3.11.1[FindBugs]
+* https://github.com/jensgerdes/sonar-pmd/releases/tag/3.2.1[PMD]
```

Comment:
Should we link exact version tags? IMHO it is better to only link and name the plugins themselves. If installed via marketplace users will install latest version. Forcing users to use a specific and probably outdated version does not make much sense for me and maintaining these versions all the time will also be tedious.
Of course a plugin update could ruin all our work in the profiles when the rule IDs have changed. This already happened and is quite a pain - but in the end we can not do pretty much about it even if we would specify exact versions here people will not consider this and install latest versions. We can only try to catch up and keep downward compatibility somehow when we import the rules.